From ad2bba54d537bced34dcc79bf356900ef9c0a301 Mon Sep 17 00:00:00 2001
From: Jakob Nixdorf <flocke@shadowice.org>
Date: Thu, 16 Apr 2015 22:09:13 +0200
Subject: [PATCH] Set the 'ar' command as variable in Makefile.tgl

Allows the location of the ar binary to be changed by passing the argument AR=/path/to/ar to make.
---
 Makefile.tgl | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/Makefile.tgl b/Makefile.tgl
index adf65d0..8312d7e 100644
--- a/Makefile.tgl
+++ b/Makefile.tgl
@@ -4,6 +4,8 @@ GENERATE_OBJECTS=${OBJ}/generate.o
 TGL_COMMON_OBJECTS=${OBJ}/tools.o
 TGL_OBJ_C=${GENERATE_OBJECTS} ${TGL_COMMON_OBJECTS} ${TGL_OBJECTS} ${TLD_OBJECTS}
 
+AR?=ar
+
 .SUFFIXES:
 
 .SUFFIXES: .c .h .o
@@ -19,7 +21,7 @@ ${OBJ}/auto/auto.o: ${AUTO}/auto.c
 	${CC} ${INCLUDE} ${COMPILE_FLAGS} -iquote ${srcdir}/tgl -c -MP -MD -MF ${DEP}/auto/auto.d -MQ ${OBJ}/auto/auto.o -o $@ $<
 
 ${LIB}/libtgl.a: ${TGL_OBJECTS} ${TGL_COMMON_OBJECTS} ${OBJ}/auto/auto.o
-	ar ruv $@ $^
+	${AR} ruv $@ $^
 
 ${EXE}/generate: ${GENERATE_OBJECTS} ${TGL_COMMON_OBJECTS}
 	${CC} ${GENERATE_OBJECTS} ${TGL_COMMON_OBJECTS} ${LINK_FLAGS} -o $@
-- 
2.3.3

